kubernetes/samples/charts/traefik/README.md

```diff
@@ -73,8 +73,8 @@ $ kubectl -n weblogic-domain2 create secret tls domain2-tls-cert --key /tmp/tls2
 # deploy the TLS IngressRoute.
 $ kubectl create -f samples/tls.yaml
 ```
-Now you can access application on the WLS domain with hostname in HTTP header.
-The loadbalancer secure port can be obtained dynamically form traefik-operator service on traefik namespace. 
+Now you can access the application on the WLS domain with hostname in HTTP header.
+The load balancer secure port can be obtained dynamically from the traefik-operator service in the traefik namespace.
```

Comment:
`traefik-operator` - put traefik-operator in code font by surrounding it in back ticks
`traefik` namespace - same with traefik

Reply:
done